core/lego_port_class.c

```diff
@@ -74,7 +74,7 @@
  * : (write-only) For modes that support it, writing the name of a driver will
  *   cause a new device to be registered for that driver and attached to this
  *   port. For example, since NXT/Analog sensors cannot be auto-detected, you
- *   must use this attribute to load the correct driver. Returns `-EOPNOTSUPP`
+ *   must use this attribute to load the correct driver (as also the I2C address). Returns `-EOPNOTSUPP`
```

Comment:
I think you mean `and` instead of `as`. Also, we could clarify that the address is needed for I2C sensors only. And I would like to keep lines under 80 chars.